generate_webdisco_target_list.js

```diff
@@ -22,9 +24,15 @@ var generateWebDiscoTargetList = function() {
     var ports = Ports.find(query).fetch();
     ports.forEach(function(port) {
       var protocol = 'http';
-      if (port.service.match(/(ssl|https)/g)) {
+      if (port.service.match(/(ssl|https)/g)){ 
         protocol = 'https';
       }
+      port.notes.forEach(function(note){
+          if(note.content.match(/SSL/))
```

Comment:
I think this may  match too much, for instance, this could match some RDP notes that get generated.